frontend/lib/src/components/shared/Modal/Modal.tsx

```diff
@@ -146,10 +146,10 @@ function Modal(props: ModalProps): ReactElement {
     },
     Dialog: {
       style: {
-        borderBottomRadius: radii.xl,
-        borderTopRadius: radii.xl,
-        borderLeftRadius: radii.xl,
-        borderRightRadius: radii.xl,
```

Comment:
This was broken since these settings do not exist. The actual setting we were using for the dialog corners was defined in baseweb and wasn't yet part of our radii set.